build/webpack.client.config.js

```diff
@@ -1,5 +1,5 @@
 const webpack = require('webpack');
-const merge = require('webpack-merge');
+const { merge } = require('webpack-merge');
```

Comment:
Breaking change in v5: https://github.com/survivejs/webpack-merge/blob/master/CHANGELOG.md#503--2020-07-06